openbas-front/src/actions/Application.js

```diff
@@ -3,18 +3,34 @@ import * as Constants from '../constants/ActionTypes';
 import * as schema from './Schema';
 import { getReferential, postReferential, putReferential, simpleCall } from '../utils/Action';
 
-export const fetchParameters = () => (dispatch) => {
-  return getReferential(schema.arrayOfParameters, '/api/settings')(dispatch);
-};
-
 export const updateParameters = (data) => (dispatch) => {
```

Comment:
rename updatePlatformParameters everywhere needed to be coherent with the others changes

Reply:
Done